app/models/concerns/waste_exemptions_engine/can_use_new_registration_workflow.rb

```diff
@@ -322,6 +322,10 @@ module CanUseNewRegistrationWorkflow
                       to: :check_registered_name_and_address_form,
                       unless: :skip_registration_number?
 
+          transitions from: :operator_postcode_form,
```

Comment:
This is unrelated to the change as described in the PR description. Was it included accidentally?

Reply:
Its been duplicated by the looks of it, I rebased the branch which had quite a lot of changes and I think its duplicated itself when I was accepting the current changes. Will remove this now :)